Avoid build-dependency on git and dependency on checking out using git.

Upstream-Status: Inappropriate [configuration]
--- scripts/build/macosx_makebundle.py.orig	2016-06-27 14:47:05.000000000 +0200
+++ scripts/build/macosx_makebundle.py	2016-06-27 14:47:26.000000000 +0200
@@ -38,7 +38,7 @@
     CONTENTS_DIR = os.path.dirname(EXE_NAME) + "/"
 CONTENTS_DIR += BUNDLE_NAME + ".app/Contents/"
 
-BUNDLE_VERSION = commands.getoutput("git --git-dir=" + SOURCE_DIR + "/.git/ describe")
+BUNDLE_VERSION = "@@VERSION@@"
 ICON_FILE = "pics/quassel.icns"
 
 
